.idea/cheval.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't need your webstorm config :smile:

You should probably be using a global gitignore that ignores your OS and IDE specific files (like `.idea/`)